dynamic-app/src/main/java/com/okta/idx/android/dynamic/auth/DynamicAuthField.kt

```diff
@@ -22,7 +22,14 @@ import androidx.lifecycle.MutableLiveData
 import com.okta.idx.android.util.emitValidation
 import com.okta.idx.kotlin.dto.IdxRemediation
 
+/**
```

Comment:
The documentation in this class is confusing to me, as it appears to talk in 2 different levels of abstraction. Can we revert the changes here, and work on those in a follow up?